includes/Documents/Invoice.php

```diff
@@ -576,6 +576,38 @@ public function get_pdf_settings_fields( $option_name ) {
 	 * UBL settings fields
 	 */
 	public function get_ubl_settings_fields( $option_name ) {
+		$extensions_available   = array();
+		$ubl_format_description = '';
+		
+		if ( ! class_exists( 'WPO_IPS_XRechnung' ) ) {
+			$extensions_available['xrechnung'] = array(
+				'title' => __( 'EN16931 XRechnung', 'woocommerce-pdf-invoices-packing-slips' ),
+				'url'   => 'https://github.com/wpovernight/wpo-ips-xrechnung/releases/latest/',
```

Comment:
I was wondering if it's OK to put a link to GitHub here. Just asking.

Reply:
I believe it simplifies the process of downloading the extension, reducing the need for users to contact us solely for that purpose.